samples/java-rest-api-clients/src/main/java/org/alfresco/java/rest/client/sample/service/RESTClientService.java

```diff
@@ -101,9 +103,9 @@ public void triggerAPAInteractions(){
 
         LOGGER.info("Task completed and process finished!!");
 
-        ResponseEntity<ListResponseContentOfCloudRuntimeEventOfobjectAndstring> ListOfAuditRawEvents = auditEventsControllerImplApi
-            .findAll(null,
-            "processInstanceId:" + processId,
+        ResponseEntity<ListResponseContentCloudRuntimeEventObjectCloudRuntimeEventType> ListOfAuditRawEvents = auditEventsControllerImplApi
```

Comment:
Looking at the changes in this file it seems we are breaking a backward compatibility. Is it fine for a patch level release? @jamalkm

Reply:
I think since this is a sample client to demonstrate REST API usage, it is ok to break backward compatibility. Anyway, we broke the backward compatibility when we released 5.2.